src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
-body {
+@import './card';
```

Comment:
It is recommended to import a separate SCSS file for variables here. Create a `_variables.scss` file to store your variables and import it at the top of this file to maintain consistency and reusability across your stylesheets.